Source/Core/Core/AchievementManager.cpp

```diff
@@ -184,7 +184,11 @@ void AchievementManager::LoadGame(const std::string& file_path, const DiscIO::Vo
   {
     std::lock_guard lg{m_lock};
 #ifdef RC_CLIENT_SUPPORTS_RAINTEGRATION
-    SplitPath(file_path, nullptr, &m_title_estimate, nullptr);
+    const auto& names = volume->GetLongNames();
+    if (names.contains(DiscIO::Language::English))
+      m_title_estimate = names.at(DiscIO::Language::English);
+    else
+      m_title_estimate = names.begin()->second;
```

Comment:
You need to have a check for the case where `names` is empty, so we don't crash. It's rare but can happen.